{{cookiecutter.project_repo}}/liccheck.ini

```diff
@@ -0,0 +1,43 @@
+# Authorized and unauthorized licenses in LOWER CASE
+[Licenses]
+authorized_licenses:
```

Comment:
what happens if the list changes, does anyone who has derived from this have to update their own liccheck?

Reply:
Yes, this is a challenge yet to be solved, if you have an ideas I would appreciate it very much

Reply:
I think that is always the issue. Even if you used this as a template project it still is only a view of what is correct at that moment in time.